src/libexpr/local.mk

```diff
@@ -26,8 +26,6 @@ $(d)/parser-tab.cc $(d)/parser-tab.hh: $(d)/parser.y
 $(d)/lexer-tab.cc $(d)/lexer-tab.hh: $(d)/lexer.l
 	$(trace-gen) flex --outfile $(libexpr_DIR)/lexer-tab.cc --header-file=$(libexpr_DIR)/lexer-tab.hh $<
 
-clean-files += $(d)/parser-tab.cc $(d)/parser-tab.hh $(d)/lexer-tab.cc $(d)/lexer-tab.hh
```

Comment:
Why this?

Reply:
So that our source tarball would include generated lexer and parser -- otherwise it gives problems with various flex/bison versions (I remember some with CentOS back when we built for it, maybe there's more).

Reply:
Our source tarball already includes the generated lexer and parser, see `dist-files` below.

Reply:
I remember the actual reason now -- it's because `make clean` isn't supposed to remove anything from a freshly unpacked tarball. It breaks Debian build for instance.